hbase-common/src/main/java/org/apache/hadoop/hbase/CellUtil.java

```diff
@@ -470,6 +470,14 @@ public static boolean matchingValue(final Cell left, final byte[] buf) {
         buf.length);
   }
 
+  /*
+    Compares tags in both cells.
+   */
+  public static boolean matchingTags(final Cell left, final Cell right) {
```

Comment:
Seems this is used by verifyReplication only.  Still better follow what other matchingXXX do

Reply:
@anoopsjohn  Thanks for the review.
I did follow the same convention that [CellUtil#matchingValue](https://github.com/apache/hbase/blob/branch-1/hbase-common/src/main/java/org/apache/hadoop/hbase/CellUtil.java#L463-L466) does.
```
  public static boolean matchingValue(final Cell left, final Cell right) {
    return Bytes.equals(left.getValueArray(), left.getValueOffset(), left.getValueLength(),
        right.getValueArray(), right.getValueOffset(), right.getValueLength());
  }
```

> Still better follow what other matchingXXX do

Anoop,
could you please elaborate what you mean ?

Reply:
Ah.. This PR is against branch-1..  I thought its for master.. My comment is applicable for master.  Better u can create PR for master first and then other branches.  Can u open PR for master also pls?  Same approach will be applicable for branch-2 also.
So as such for this PR +1

Reply:
Tags are little bit convoluted in master branch. There are 2 implementation for KeyValue there..one backed by ByteBuffer and other backed by Array. We wanted this in our internal branch which is based of branch-1.6 so I created first again branch-1. Wanted to get a quick feedback from community first.

@anoopsjohn  Will create a PR against master soon.